### PR TITLE
hatch: add page

### PR DESCRIPTION
**Pull request by joweich**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/hatch.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Build Hatch project:
+
+`hatch build`
+
+- Remove build artifacts:
+
+`hatch clean`
+
+- Create default environment with dependencies defined in pyproject.toml:
```
Comment (spageektti) [inline pages/common/hatch.md:23]:
```suggestion
- Create default environment with dependencies defined in the `pyproject.toml` file:
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution

**Review by karthik-script — APPROVED** [review]
LGTM

File: pages/common/hatch.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`hatch clean`
+
+- Create default environment with dependencies defined in the `pyproject.toml` file:
+
+`hatch env create`
+
+- Show environment dependencies as table:
+
+`hatch dep show table`
```
Comment (acuteenvy) [inline pages/common/hatch.md:29]:
```suggestion
- Create a new Hatch project:

`hatch new {{project_name}}`

- Initialize Hatch for an existing project:

`hatch new --init`

- Build a Hatch project:

`hatch build`

- Remove build artifacts:

`hatch clean`

- Create a default environment with dependencies defined in the `pyproject.toml` file:

`hatch env create`

- Show environment dependencies as a table:

`hatch dep show table`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**